docs/tutorials/pi/make-a-plant-watering-robot.md

```diff
@@ -0,0 +1,483 @@
+
+        # create the cs (chip select)
+        cs = digitalio.DigitalInOut(board.D25) # TODO: Need to check this: example was D5. Documentation is confusing.
+
+        # create the mcp object
+        mcp = MCP.MCP3008(spi, cs)
+
+        # create an analog input channel on pin 0
+        chan = AnalogIn(mcp, MCP.P0)
+
+        while x<10:
+            # read adc channel 0
+            reading = chan.value
+            input.append(reading)
+            x+=1
+
+        return input
+
+async def main():
+   srv = Server(components=[MoistureSensor("moisture_sensor")])
+   await srv.serve()
+
+if __name__ == "__main__":
+   asyncio.run(main())
+```
+
+You can modify this example code as necessary.
+
+Now, go back to your robot's page on [the Viam app](https://app.viam.com) and navigate to the **CONFIG** tab, then to the **REMOTES** sub-tab.
+
+Add your sensor server as a remote part, called `my-sensor-server`:
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+<img src="../../img/plant-watering-pi/sensor-remote-config-builder.png" alt="Creation of a custom sensor remote in the Viam app config builder." style="max-width:800px" />
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+// "components": [ board & motor ... ],
+"remotes": [
+    {
+      "name": "my-sensor-server",
+      "address": "localhost:9090"
+    }
+]
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+<br>
+
+Then, navigate to the **PROCESSES** subtab and create a process called `run-sensor-server`:
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+<img src="../../img/plant-watering-pi/sensor-process-config-builder.png" alt="Creation of a pump motor in the Viam app config builder." style="max-width:800px" />
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  // "components": [ board & motor ... ] ,
+  // "remotes" [ my-sensor-server ... ] ,
+  "processes": [
+    {
+      "id": "run-sensor-server",
+      "log": true,
+      "name": "sudo",
+      "args": [
+        "-u",
+        "<my_username>",
+        "python",
+        "sensor.py"
+      ],
+      "cwd": "/home/<my_username>/plant-watering-robot"
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Add Python Control Code
+
+Follow these instructions to start working on your Python Control Code:
+
+1. Navigate to your robot's page in [the Viam app](https://app.viam.com), and click on the **CODE SAMPLE** tab.
+Follow the instructions in this tab.
+2. Click **COPY CODE** to copy a code sample that establishes a connection with your robot when run.
+3. Paste this code sample into a new file in your `plant-watering-robot` directory.
+4. Name the file <file>plant-watering-robot.py</file>, and save it.
+  
+For example, run the following commands to create and open the file:
+
+``` shell
+cd plant-watering-robot
+touch plant-watering-robot.py
+vim plant-watering-robot.py
+```
```

Comment:
I don't think this is actually helpful here. You give instructions above and people will use the tools they are familiar with
```suggestion
```